sys/vane/jael.hoon

```diff
@@ -1673,11 +1786,69 @@
   ^-  (unit (unit cage))
   :: XX security
   ?.  =(lot [%$ %da now])  ~
-  %-  some
-  ?.  =(%$ ren)  ~
-  %+  bind  (~(scry of [now eny] lex) syd tyl)
-  =-  ~!  -  -
-  |=(a/gilt [-.a (slot `@`3 !>(a))])
+  :: %-  some
+  ?.  =(%$ ren)  [~ ~]
+  :: =-  ~!  -  -
+  :: %+  bind  (~(scry of [now eny] lex) syd tyl)
+  :: |=(a/gilt [-.a (slot `@`3 !>(a))])
+  ?+    syd
+      ~
+  ::
+      %life
+    ?.  ?=([@ ~] tyl)
+      ~
+    ?.  ?&  ?=(%& -.why)
+            (~(has by pry.urb.lex) p.why)
+        ==
+      ~
+    =/  who  (slaw %p i.tyl)
+    ?~  who  ~
+    ?:  =(u.who p.why)
+      ``[%atom !>(lyf.own.sub.lex)]
+    =/  pub  (~(get by kyz.puk.sub.lex) u.who)
+    ?~  pub  [~ ~]
+    ``[%atom !>(life.u.pub)]
+  ::
+      %earl
```

Comment:
Any reason we're not deviating from archaic terminology and just saying `moon` instead?

Reply:
Consistency over clarity.

Reply:
One of these days someone will make a PR for something like "gaxy, star, lant, moon, come". It may or may not be me. We may or may not spend a couple months bikeshedding about various four-letter versions and just using the full names.  
imo the `rank` terminology needs to go sooner rather than later, but the consistency point is fair, and this is not a discussion that needs to happen in/for PR.